apps/lasp/src/lasp_core.erl

```diff
@@ -337,6 +337,26 @@ bind(Id, Value, Store) ->
 
 %% @doc Define a dataflow variable to be bound a value.
 -spec bind(id(), value(), function(), store()) -> {ok, var()}.
+bind(Id, {delta, Value}, MetadataFun, Store) ->
```

Comment:
This is still going to result in the entire state being transmitted by the broadcast backend, so we need to figure out a way in the broadcast backend to only send the delta.  We probably need to change the return value from this function to return only the delta, so the broadcast backend uses that for transmission.

Make sense?